src/main/java/bobby/ui/Events.java

```diff
@@ -0,0 +1,48 @@
+package bobby.ui;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an event task that occurs within a specific time range.
+ * An Event task has a description, a start time, and an end time.
+ */
+public class Events extends Task{
+
+    /**
+     * The start time of the Event
+     */
+    protected LocalDateTime start;
+
+    /**
+     * The end time of the Event
+     */
+    protected LocalDateTime end;
```

Comment:
Should your start and end variables be classified together under one block instead?